examples/basic.html

```diff
@@ -66,6 +66,7 @@ <h2>
       </pre>
       <p>
         The <a>Foo</a> interface is nice. Lets you do stuff.
+        SEE . <a href="#intro"></a>
```

Comment:
Please remove this change. Instead, modify tests/spec/core/section-refs-spec.js, line 14. You can see why that test failed by looking at the TravisCI logs.

Reply:
@marcoscaceres all requested change are done.